articleEditor/editor.js

```diff
@@ -0,0 +1,346 @@
+
+    if (!/[._!@0-9-a-zA-Zа-яА-Я]{2,}/.test(window.valuesChecked.articleName)) {
+        warning(`Некорректный ввод! Заполните поле - Название!`);
+    } else if (window.valuesChecked.articleDescription.length > 49) {
+        return window.valuesChecked;
+    } else {
+        warning(`Краткое описание должно содержать от 50 символов!`);
+    }
+}
+function CarouselMaket(array_articles) {
+
+    document.getElementsByClassName(`carousel-indicators`)[0].innerHTML = "";
+    document.getElementsByClassName(`carousel-inner`)[0].innerHTML = "";
+
+    for (let el in array_articles) {
+        let li = document.createElement(`li`);
+
+        li.setAttribute(`data-target`, `#sliderFade`);
+        li.setAttribute(`data-slide-to`, `${el}`);
+        document.getElementsByClassName(`carousel-indicators`)[0].appendChild(li);
+        let div = document.createElement(`div`);
+        div.className = "carousel-item";
+
+        div.innerHTML = `<div class="alley">
+                <div class="mt-4 mb-5">
+                <div class="h2">${array_articles[el].articleName}</div>
+                    <a class="float-left rounded  p-1 font-weight-bold badge-secondary mr-3">${array_articles[el].articlenickName}</a>
+                    <a class="badge pt-2">Last update ${updateDate(array_articles[el].articleLastDate)}</a>
+            </div>
+            ${array_articles[el].articleContent}
+         </div>`;
+        document.querySelector(`.carousel-inner`).appendChild(div);
+    }
+
+    let a = document.createElement(`a`);
+    a.className = "close closeArticles";
+    a.innerHTML = "[x]";
+    document.getElementsByClassName(`carousel-inner`)[0].insertAdjacentElement(`afterbegin`, a);
+    document.querySelector(`.closeArticles`).addEventListener(`click`, () => {
+
+        for (let el of document.querySelectorAll(`[data-slide-to]`)) {
+
+            if (el.className.indexOf(`active`) != -1) {
+                el.classList.toggle(`active`);
+                break;
+            }
+        }
+
+        document.getElementsByClassName(`carousel-item active`)[0].classList.toggle(`active`);
+        document.getElementsByClassName(`cardsContainer`)[0].classList.toggle(`hidden`);
+
+        if(document.getElementById(`SearchArticle`).value.replace(/\s+/g, ' ').trim()){
+            return;
+        }
+        ShowArticles(window.articlesFromLS);
+        reattachListeners();
+
+    });
+}
+function inputValues(values) {
+
+    let ckeditor = CKEDITOR.instances.articleContent,
+        name = document.getElementById('articleName'),
+        description = document.getElementById('articleDescription');
+    if (values) {
+
+        ckeditor.setData(values.articleContent);
+        name.value = values.articleName;
+        description.value = values.articleDescription;
+    } else {
+        ckeditor.setData("");
+        name.value = "";
+        description.value = "";
+    }
+}
+function EditDelete(event) {
```

Comment:
EditDeleteAddReplaceRemoveShouldChange...
Use proper method names. Also, make decomposition to each `case` - all it should be in separate functions.